web/src/main/angular/src/app/core/components/transaction-id-search/transaction-id-search-container.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="l-search-group-wrap font-opensans">
+    <input type="text" placeholder="search pTxId" class="l-search-input"
+           [(ngModel)]="inputValue" (keydown.esc)="onClear()" (keyup.enter)="onSearchId()">
+    <button class="l-search-button fas fa-search" (click)="onSearchId()"></button>
```

Comment:
Please use `ppSearchInput` directive. It has already taken care of a couple of `Keyboard` events. 
```suggestion
<input 
    type="text"
    ppSearchInput
    placeholder="search pTxId"
    class="l-search-input"
    [useEnter]="searchUseEnter"
    (outSearch)="onSearchId($event)"
    #searchInput>
<button class="l-search-button fas fa-search" (click)="onSearchId(searchInput.value.trim())"></button>
```


You can refer to other use cases in other components.